src/config.js

```diff
@@ -43,6 +44,9 @@ const deleteBranchAfterMerge =
 
 const dry = MERGERINE_DRY === 'true' || config.dry
 
+const mergeableStateCheck =
+  MERGERINE_MERGEABLE_STATE_CHECK === 'false' || config.mergeableStateCheck
```

Comment:
Shouldn't this check for `MERGERINE_MERGEABLE_STATE_CHECK === 'true'`? Was your intent to enable it by default? If so, perhaps it should've been:
```js
MERGERINE_MERGEABLE_STATE_CHECK ?
  MERGERINE_MERGEABLE_STATE_CHECK !== 'false' : 
  config.mergeableStateCheck
```

Reply:
Fixed this, with new unit tests to prove it, in 2320a29b8292484223a4dd13b650389b79121271, published with `1.10.1`.